gatsby-node.js

```diff
@@ -48,7 +48,7 @@ exports.onCreateNode = async ({ node, getNode, getNodesByType, actions, createNo
     }
     return;
   }   
-  const basePath = category === 'user' ? `docs/en/userguide` : `en`;
+  const basePath = category === 'user' ? `docs/en/userguide` : thirdparty ? `docs/en` : `en`;
```

Comment:
```suggestion
  const basePath = category === 'user' ? 'docs/en/userguide' : (thirdparty ? 'docs/en' : 'en');
```
I'm fairly sure this change makes no difference (it shouldn't :-) though just make this clearer since there's a double ternary

Also change the backticks to single quotes

Reply:
Done